<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>продукт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>масса</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>новая рецептура</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="old_radio_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>старая рецептура</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="recipe_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="old_recipe_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="product_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mass_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>приготовить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>630</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
